projects/elife.yaml

```diff
@@ -205,26 +205,34 @@ defaults:
             address: https://master-server.elifesciences.org:8200
     aws-alt:
 
-        # note! all aws-alt configurations below will be inherited by all projects.
+        # note! all aws-alt configurations below will be inherited by *all* projects.
+        # note! only use lowercase and hyphens for alt-config names.
 
         fresh:
             description: uses a public Ubuntu AMI instead of an elife generated basebox.
             ec2:
                 # should be the same as basebox '18.04' and 'standalone18.04' aws-alt configurations
                 ami: ami-0b425589c7bb7663d # bionic, build 20180814, hvm:ebs-ssd
 
-        # deprecated, use `sNextSalt` instead
+        # deprecated, use `s-next-salt` instead
```

Comment:
My main constraint when using e.g. `s1804` was length of hostnames that get generated, for example `pr-27-base-update--personalised-covers.elifesciences.org` or similar. I'd try to keep it as short as possible, maybe `snextsalt`? `snsalt`?

Reply:
> snsalt

ha. snail salt. I can live with that